src/components/landing-page/UseYourProfile/UseYourProfile.tsx

```diff
@@ -1,18 +1,14 @@
-import { authOptions } from '@/app/(auth)/auth'
+import { getAuthorizedUser } from '@/app/(auth)/helpers'
 import MyProfileBtn from '@/app/(profile)/(components)/MyProfileBtn/MyProfileBtn'
 import CreateProfileBtn from '@/app/(profile)/my-profile/(components)/CreateProfileBtn/CreateProfileBtn'
 import { findProfileWithUserInclude } from '@/backend/profile/profile.service'
-import { getServerSession } from 'next-auth'
 import Image from 'next/image'
 import UseYourProfileImg from '../../../assets/images/UseYourProfile.png'
 import style from './UseYourProfile.module.scss'
 
 const UseYourProfile = async () => {
-  const session = await getServerSession(authOptions)
-
-  const profile = session
-    ? await findProfileWithUserInclude(session.user.email)
-    : null
+  const { user } = await getAuthorizedUser()
```

Comment:
**question:** What about moving user to session object? Do we want to split in different ticket?

Reply:
This implementation has been added at https://github.com/nerdbord/good-dev-hunting-app/pull/263/commits/bc5586fbfed1a0473c20998a51af006fa9ab7330. It's working fine now. Our next step is to verify if there are any remaining queries to the database in other components.

Reply:
done in https://github.com/nerdbord/good-dev-hunting-app/pull/263/commits/db88700f82eec9b896628c3f671298b5714baa82